scripts/library.sh

```diff
@@ -146,6 +146,24 @@ function wait_until_service_has_external_ip() {
   return 1
 }
 
+# Waits for the endpoint to be routable
+# Parameters: $1 - External ingress IP address `kubectl get svc knative-ingressgateway -n istio-system -o jsonpath="{.status.loadBalancer.ingress[*].ip`
```

Comment:
The kubectl commands are unnecessary, it's up to the caller to figure them out. You can, however, make the IP optional, which then would default to the result of the `kubectl` command above.

Reply:
removed the commands.